core/crypto/poseidon_hash_test.go

```diff
@@ -0,0 +1,26 @@
+package crypto_test
+
+import (
+	"testing"
+
+	"github.com/NethermindEth/juno/core/crypto"
+	"github.com/NethermindEth/juno/core/felt"
+)
+
+func BenchmarkPoseidon(b *testing.B) {
```

Comment:
Please try to use [`testing.TB`](https://pkg.go.dev/testing#TB) interface

Reply:
doesnt work for some reason, requires a wrapper of some sort.